contrib/lock/client/client.go

```diff
@@ -66,6 +66,7 @@ func write(key string, value string, version int64) error {
 	}
 
 	respBytes, err := io.ReadAll(httpResp.Body)
+	httpResp.Body.Close()
```

Comment:
Please put this line at line 73 (after the `if err != nil {} `).  When err != nil, usually means the response body has already closed.

This comment applies to following changes as well.